### PR TITLE
Add xorg-libxrandr

### PR DESCRIPTION
**Pull request by Xarthisius**
It's a dependency for packages I'm planning to add, but also for packages already in conda-forge (e.g. freeglut). Not sure, why the latter compiles without it...

### CODE REVIEW

**Comment by ngoldbaum** [comment]
ping @jakirkham

File: recipes/xorg-libxrandr/build.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+./configure "${configure_args[@]}"
+make -j$CPU_COUNT
+make install
+make check
+rm -rf $uprefix/share/man $uprefix/share/doc/${PKG_NAME#xorg-}
+
+# Non-Windows: prefer dynamic libraries to static, and dump libtool helper files
+if [ -z "VS_MAJOR" ] ; then
+    for lib_ident in Xrender; do
+        rm -f $uprefix/lib/lib${lib_ident}.la $uprefix/lib/lib${lib_ident}.a
```
Comment (epruesse) [inline recipes/xorg-libxrandr/build.sh]: This should be adapted to xrandr

File: recipes/xorg-libxrandr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
```
Comment (epruesse) [inline recipes/xorg-libxrandr/meta.yaml]: Technically already default with current conda-build

File: recipes/xorg-libxrandr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+about:
+  home: https://www.x.org/
+  license: MIT
+  license_family: MIT
+  license_file: COPYING
+  summary: 'The X.org Xrandr library'
+
+extra:
+  recipe-maintainers:
+    - Xarthisius
```
Comment (epruesse) [inline recipes/xorg-libxrandr/meta.yaml]: Ask @pkgw whether he wants to be co-maintainer on this?

**Comment by Xarthisius** [comment]
@pkgw would you like to co-maintain?

**Comment by pkgw** [comment]
Sure, I can be added as a co-maintainer. This can be done after merging, though, if you don't want to wait for the CI to rerun.

Otherwise, the recipe looks good to me. If no one else has any comments I can merge.

**Pull request merged**